Raw API calls are not supported, since they are disabled in the Makefile 
for a fix to bug 22535919 which requires API calls to be disabled.
Removes the raw API functions if RAW API is not allowed.
Will attempt to send upstream.

--- libffi-3.4.4/include/ffi.h.in
+++ libffi-3.4.4/include/ffi.h.in
@@ -296,6 +296,7 @@ typedef ffi_raw ffi_java_raw;
 #endif
 
 
+#if !FFI_NO_RAW_API
 FFI_API
 void ffi_raw_call (ffi_cif *cif,
 		   void (*fn)(void),
@@ -305,6 +306,7 @@ ffi_raw *avalue);
 FFI_API void ffi_ptrarray_to_raw (ffi_cif *cif, void **args, ffi_raw *raw);
 FFI_API void ffi_raw_to_ptrarray (ffi_cif *cif, ffi_raw *raw, void **args);
 FFI_API size_t ffi_raw_size (ffi_cif *cif);
+#endif
 
 /* This is analogous to the raw API, except it uses Java parameter
    packing, even on 64-bit machines.  I.e. on 64-bit machines longs
--- libffi-3.4.4/libffi.pc.in
+++ libffi-3.4.4/libffi.pc.in
@@ -8,4 +8,4 @@ Name: @PACKAGE_NAME@
 Description: Library supporting Foreign Function Interfaces
 Version: @PACKAGE_VERSION@
 Libs: -L${toolexeclibdir} -lffi
-Cflags: -I${includedir}
+Cflags: -I${includedir} -DFFI_NO_RAW_API
